assets/js/components/setup.js

```diff
@@ -35,8 +35,8 @@ import Button from '../components/button';
 import Layout from '../components/layout/layout';
 import data, { TYPE_CORE } from '../components/data';
 import { trackEvent, clearWebStorage, getSiteKitAdminURL } from '../util';
-import STEPS from '../components/setup-wizard/wizard-steps';
-import WizardProgressStep from '../components/setup-wizard/wizard-progress-step';
+import STEPS from '../components/legacy-setup/wizard-steps';
+import WizardProgressStep from '../components/legacy-setup/wizard-progress-step';
```

Comment:
The imports here can be simplified to be relative to the `assets/js/components` directory, similar to the previous comment. Would be good to also fix that for the untouched imports above where this is also unnecessary.